bookkeeper-server/src/test/java/org/apache/bookkeeper/client/api/BKExceptionTest.java

```diff
@@ -38,15 +37,14 @@ public void testGetMessage() throws Exception {
                 int code = f.getInt(null);
                 String msg = BKException.getMessage(code);
                 if (code == UnexpectedConditionException) {
-                    assertThat(msg, equalTo("Unexpected condition"));
+                    assertEquals(msg, "Unexpected condition");
```

Comment:
nit: expected message should be the first parameter: assertEquals("Unexpected condition", msg);